### ISSUE TITLE
Documentation for to filtering traces

### ISSUE THREAD

**Issue by moonchen** [ARMmbed/mbed-trace#83]
It looks like the documentation for filtering of traces is missing from the README.

**Comment by jupe** [comment]
Good point. [Code documentation](https://github.com/ARMmbed/mbed-trace/blob/6df2572dfb803596049b87fbad4d761acaef69c3/mbed-trace/mbed_trace.h#L266) is in place but there should be documentation also in README.

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTSYST-2940
